Add notes about QAUTOTUNE and AUTOTUNE for copter

Hwurzburg opened the pull request:
notes about QAUTOTUNE not be applicable for Tailsitter pitch and notes that yaw tune cannot loose position hold during tune

Hwurzburg commented:
@rmackay9 @tridge  review request bump

IamPete1 commented on plane/source/docs/qautotune-mode.rst:
```diff
@@ -4,7 +4,7 @@
 =========
 
-As of Plane-4.0, autotuning PIDs in QuadPlane is supported with this flight mode. This uses the same system as the AUTOTUNE mode for copter. It should allow you to improve the tune without having to manually adjust PIDs.
+As of Plane-4.0, autotuning PIDs in QuadPlane is supported with this flight mode. This uses the same system as the AUTOTUNE mode for copter. It should allow you to improve the tune without having to manually adjust PIDs. It is not recommended to use QAUTOTUNE for Tailsitter pitch axis, or any axis which requires feed forward pid contributions.
```
```suggestion
As of Plane-4.0, autotuning PIDs in QuadPlane is supported with this flight mode. This uses the same system as the AUTOTUNE mode for copter. It should allow you to improve the tune without having to manually adjust PIDs. It is not recommended to use QAUTOTUNE for Tailsitter pitch or yaw axis, or any axis which requires feed forward pid contributions.
```

IamPete1 commented on plane/source/docs/quadplane-vtol-tuning-process.rst:
```diff
@@ -372,6 +372,7 @@ There a number of problems that can prevent QAUTOTUNE from providing a good tune
 - Non-linear ESC response.
 - Very low setting for :ref:`Q_M_SPIN_MIN <Q_M_SPIN_MIN>`.
 - Overloaded propellers or motors.
+- Tailsitter's pitch axis, since it requires feed-forward.
```
```suggestion
- Tailsitter's pitch and yaw axis, since it requires feed-forward.
```

tridge commented on copter/source/docs/autotune.rst:
```diff
@@ -83,7 +81,7 @@ If the vehicle feels sloppy after the AutoTune, try increasing the :ref:`AUTOTUN
 ==================================
 
-In Copter-3.5 (and higher) AutoTune performs a weak position hold if invoked from Loiter or PosHold flight modes (as opposed to AltHold).
+In Copter-3.5 (and higher) AutoTune performs a weak position hold if invoked from Loiter or PosHold flight modes (as opposed to AltHold) while doing a pitch or roll autotune (not yaw).
```
are you sure it doesn't do it in yaw? It should.I've just tested and it does do weak pos hold in yaw autotune

Hwurzburg replied:
at this point no....fixing...

tridge commented on plane/source/docs/qautotune-mode.rst:
```diff
@@ -27,4 +27,6 @@ You can reposition while tuning by moving your sticks. Once you re-center the st
 
 You can read more about AutoTune for copters here: :ref:`autotune` . The autotune for quadplanes works the same way.
 
+.. note:: Like Copter, entering QAUTOTUNE from QLOITER will attempt a loose position hold. However, this does not apply for yaw tunes and the vehicle will drift with the wind during yaw axis tuning.
```
it does apply to yaw

rmackay9 replied:
yes, for copter as well it will hold position during yaw tunes

Hwurzburg replied:
I must have imagined it....I did not let it drift long, when I thought it got outside the hold limits I stopped the flight......will fix this....sorry for the red herring

Hwurzburg commented:
removed the yaw position hold stuff...the copter stuff is just updating the aux functions comments, and I have also added that yaw is not be done in QAUTOTUNE in a few other places that Pete had missed in his review....pushing...will merge tomorrow after I look it  over once more...
